docs/Course Wiki/CS Course Offerings/CS173.md

```diff
@@ -0,0 +1,70 @@
+
+## Prerequisites
+
+- [CS124](./CS124.md) OR [ECE220](../ECE%20Course%20Offerings/ECE220.md)
+- MATH220 OR [MATH221](../MATH%20Course%20Offerings/MATH221.md)
+
+CS 173's few prerequisites are of limited importance to being able to succeed in the course. Coding experience is mainly helpful for analyzing Big-O for several algorithms but also helps in the specific mindset it promotes. Having the prerequisites done is still recommended, especially since the requirement for ECE students to take ECE 220 before this course has been enforced recently.
+
+## When to Take it
+
+Due to this class being a hard prerequisite for [CS 225](./CS225.md), we now recommend taking MATH 213 concurrently with ECE 220 instead.
+
+Discrete Structures officially a prerequisite for CS 225, which is the gateway to most, if not all CS courses.  As such, take Discrete Structures as early as possible, especially for Computer Engineering majors.  Most higher-level CS courses will definitely cover discrete math concepts so the topics introduced in CS 173 will be reviewed again and again.  Furthermore, a lot of the computer engineering concepts are also introduced in this class and so it will complement a lot of early computer engineering classes, like ECE 120 and ECE 220.  Though it is possible to do well even when taking discrete structures concurrently with a coding class, like ECE 220 or CS 125, it is highly recommended to take the coding class before taking CS 173.  Exposure to coding will greatly help the understanding of discrete math and taking them concurrently will most likely just be a bigger burden to try and grasp the concepts faster. 
+
+## Course Structure
+
+### Lecture
+
+CS173 has 2-5 asynchronous lecture videos per week which are posted on the course website.  Aside from this, lecture notes and sections to read are also posted, which go over the topics covered in the videos.
+
+### Tutorial and Homework
+
+CS173 has tutorial sections similar to discussion sections of other classes.  They are held in a lecture hall and nothing is required to be turned in during class.
+
+Each week will have one Pre-Tutorial Checkpoint due the day before tutorial, and one Homework due 4 days after tutorial. These are autograded on PrairieLearn, and you can submit them as many times as needed to get full credit.
+
+The Pre-Tutorial Checkpoints are simpler questions designed to make sure that you have completed the readings and watched the lecture videos, and the Homeworks released after tutorial go into more depth.
+
+### Exams and Grading
+
+CS173 has 6 smaller Examlets and a final, and typically does not curve exam scores. Exam scores make up roughly 90% of a student's grade and the remainder comes from pretutorial checkpoints and homework.
+
+They'll drop your lowest checkpoint grade and your lowest homework grade when computing your final averages (though remember you can submit these multiple times, so it should not be too difficult to ultimately get 100% on all of them).
```

Comment:
They'll drop ... -> The lowest checkpoint grade and the lowest homework grade are dropped when computing final averages.

Info about multiple submissions can be moved out of () and into its own sentence